Makefile

```diff
@@ -95,11 +95,14 @@ ALL_CXXFLAGS = -std=gnu++11 -fno-rtti
 
 # Determine pointer size in bits.
 BITS := $(bits)
+UNAME_M := $(shell uname -m)
 
 ifeq ($(BITS),64)
+ifneq ($(UNAME_M),aarch64)
```

Comment:
from looking at the rest of the file, i believe we should be indenting this and the corresponding `endif`